### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This bundle provides caching services
 
 Branch | Travis | Coveralls |
 ------ | ------ | --------- |
-3.x   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
+2.x   | [![Build Status][travis_stable_badge]][travis_stable_link]     | [![Coverage Status][coveralls_stable_badge]][coveralls_stable_link]     |
```
Comment (OskarStark) [inline README.md]: @greg0ire this looks wrong to me, do wen need some update on dev-kit?

Comment (kunicmarko20) [reply README.md]: yep, it is wrong, @greg0ire changed this yesterday but dev-kit reverted it 😂

Comment (greg0ire) [reply README.md]: Told you it was going to mess things up ^^

Comment (OskarStark) [reply README.md]: hehe ok

**Pull request merged**